src/objects/Lotto.js

```diff
@@ -0,0 +1,69 @@
+import { ERROR_MESSAGE, LOTTO_RANK_MAP } from '../lib/constants.js';
+import {
+  getIsAllItemsUnique,
+  getIsArrayLengthMatch,
+  intersection,
+} from '../lib/utils.js';
+
+class Lotto {
+  #numbers;
+
+  constructor(numbers) {
+    this.#numbers = numbers;
+
+    this.#validate();
+    this.#sortByAscending();
+  }
+
+  get numbers() {
+    return this.#numbers;
+  }
+
+  getRank(winningNumbers, bonusNumber) {
+    const winningCount = this.#getWinningNumberCount(winningNumbers);
+    const isBonusMatch = this.#getIsBonusMatch(bonusNumber);
+
+    const rank = Lotto.#calculateRank(winningCount, isBonusMatch);
+    return rank;
+  }
+
+  #validate() {
+    if (!getIsArrayLengthMatch(this.#numbers, 6))
```

Comment:
만드신 상수를 사용하시면 더 좋을 것 같아용 
```suggestion
    if (!getIsArrayLengthMatch(this.#numbers, LOTTO.NUMBER_COUNT))
```

Reply:
꼼꼼하십니다 👍👍